assets/sendmail_template.txt

```diff
@@ -14,7 +14,7 @@ Content-Transfer-Encoding: base64
 Content-ID: <nfcorepipelinelogo>
 Content-Disposition: inline; filename="nf-core-rnaseq_logo.png"
 
-<% out << new File("$baseDir/assets/nf-core-rnaseq_logo.png").
+<% out << new File("nf-core-rnaseq_logo.png").
   bytes.
```

Comment:
Modification to address the same issue as described [here](https://github.com/TheJacksonLaboratory/atacseq/pull/3/files#r419120532)